Add `source` attribute to `dbt_task` and `sql_task.file` tasks to support files from workspace (#3208)
databricks/terraform-provider-databricks: docs/resources/job.md

### dbt_task Configuration Block

* `commands` - (Required) (Array) Series of dbt commands to execute in sequence. Every command must start with "dbt".

> * `source` - (Optional) What is the source of this project. Possible values are `WORKSPACE` and `GIT`.  Default value depends on if `git_source` block presents in the job definition or not - if it's present, then the default value is `GIT`.

Reviewer: ```suggestion
* `source` - (Optional) The source of the project. Possible values are `WORKSPACE` and `GIT`.  Defaults to `GIT` if a `git_source` block is present in the job definition.
```

> * `project_directory` - (Optional  if `source` is `GIT`, Required when `source` is `WORKSPACE`) If `source` is `GIT`, then it should be a relative path to the directory in the repository specified in `git_source` where dbt should look in for the `dbt_project.yml` file. If not specified, defaults to the repository's root directory. Equivalent to passing `--project-dir` to a dbt command.  If `source` is `WORKSPACE`, then it should be absolute path to the folder in the workspace.

Reviewer: ```suggestion
* `project_directory` - (Required when `source` is `WORKSPACE`) The path where dbt should look for `dbt_project.yml`. Equivalent to passing `--project-dir` to the dbt CLI. 
  * If `source` is `GIT`: Relative path to the directory in the repository specified in the `git_source` block. Defaults to the repository's root directory when not specified.
  * If `source` is `WORKSPACE`: Absolute path to the folder in the workspace.
```

  * `alert_id` - (Required) (String) identifier of the Databricks SQL Alert.
  * `subscriptions` - (Required) a list of subscription blocks consisting out of one of the required fields: `user_name` for user emails or `destination_id` - for Alert destination's identifier.
  * `pause_subscriptions` - (Optional) flag that specifies if subscriptions are paused or not.
* `file` - (Optional) block consisting of single string fields: 

>   * `source` - (Optional) What is the source of this project. Possible values are `WORKSPACE` and `GIT`.

Reviewer: ```suggestion
  * `source` - (Optional) The source of the project. Possible values are `WORKSPACE` and `GIT`.
```

>   * `path` - If `source` is `GIT`, then it a relative path to the file (inside the Git repository) with SQL commands to execute.  If `source` is `WORKSPACE` then it should be absolute path to the file in the workspace.

Reviewer: ```suggestion
  * `path` - If `source` is `GIT`: Relative path to the file in the repository specified in the `git_source` block with SQL commands to execute. If `source` is `WORKSPACE`: Absolute path to the file in the workspace with SQL commands to execute.
```
